### DISCUSSION TITLE
Questions about the impact energy for EI mode

### DISCUSSION THREAD

**Discussion by ghost** [qcxms/QCxMS#77] [Q&A] [answered]
- What exactly is the parameter `eimpw`, impact energy distribution. Given the IEE distribution of either the poisson or gauss type, will `eimpw` uniquely defines the distribution? If yes, can you please provide some formulas?
- What is the better distribution type for the impact energy distribution: Gaussian or Poisson? Assuming that fragmentation is a rare event, it makes sense to model the impact energy distribution with a Poisson distribution. However, there are many different channels (fragmentation, internal conversion, and vibrational excitation) into which the excess energy can flow. Gaussian distribution would make sense when fragmentation is not rare because for large enough fragmentation rate, the Poisson distribution can be approximated by a Gaussian based on Central Limit Theorem.
- I'm a bit confused why `eimpw=0.1 eV` and `ieeatm=0.6 eV/atom` are the default values. Somehow there is more IEE/atom than available impact energy.

**Comment by JayTheDog** [comment, marked as answer]
- For changing the default values, please refer to the manual at https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms.html .
- For an intensive discussion on the different settings and their influences on the EI spectrum (IEE width etc), please refer to the original publication (especially check the SI, where a lot of there things were discussed) https://onlinelibrary.wiley.com/doi/10.1002/anie.201300158
- `eimpw` is **not** the impact energy, but the width of the energy distribution. `eimp0` is the impact energy. Please refer to the manual
